packages/@okta/vuepress-site/docs/guides/oie-upgrade-mfa-enroll-policy/main/index.md

```diff
@@ -9,34 +9,43 @@ meta:
 
 ## Overview
 
-With Okta Identity Engine, the definition of factors and authenticators have been differentiated to align with industry standards. Identity Engine uses authenticators in its authenticator enrollment policy settings, whereas Okta Classic Engine uses factors in its Multifactor (MFA) enrollment policy settings.
+Okta Identity Engine sets apart factors and authenticators to align with industry standards:
```

Comment:
```suggestion
Identity Engine sets apart factors and authenticators to align with industry standards:
```

Reply:
@susanharper-okta This is the first mention of Identity Engine on the page.